assets/styles/common.css

```diff
@@ -1,5 +1,5 @@
 @import "https://fonts.googleapis.com/css?family=Lato:400,400i|Source+Sans+Pro|Sriracha&display=swap";
-@import "./button.css";
+/*@import "./button.css";*/
```

Comment:
устранено, но в консоли тогда ошибка: Refused to apply style from 'http://localhost:63342/assets/styles/button.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.